### ISSUE TITLE
The 0.10.1 update cover the file what I edited the snippets.json in the Emmet. So I need to edit it again. Is it can improve?

### ISSUE THREAD

**Issue by karl20888** [Microsoft/vscode#157]
The 0.10.1 update cover the file what I edited the snippets.json in the Emmet. So I need to edit it again. Is it can improve?

**Comment by joaomoreno** [comment]
This is very hard to understand, can you maybe rephrase?

**Comment by karl20888** [comment]
@joaomoreno has modified the default snippets. After he updated the vs code to version 0.10.1, he lost the modification. He suggests that could he overwrite the default snippets using "User" or "Workplace" snippets, just like the way we overwrite the default settings.

**Comment by aeschli** [comment]
dup of #154
